src/overlays/actors/ovl_En_Fg/z_en_fg.c

```diff
@@ -461,26 +471,27 @@ void EnFg_DrawDust(PlayState* play, EnFgEffectDust* dustEffect) {
     Gfx_SetupDL25_Xlu(play->state.gfxCtx);
 
     for (i = 0; i < 10; i++, dustEffect++) {
-        if (dustEffect->type) {
-            if (!firstDone) {
-                POLY_XLU_DISP = Gfx_SetupDL(POLY_XLU_DISP, SETUPDL_0);
-                gSPDisplayList(POLY_XLU_DISP++, object_fr_DL_00B328);
-                gDPSetEnvColor(POLY_XLU_DISP++, 0, 0, 0, 0);
-                firstDone = true;
-            }
+        if (dustEffect->type == 0) {
+            continue;
+        }
 
-            if (1) {}
-            alpha = (255.0f / 16.0f) * dustEffect->timer;
-            gDPSetPrimColor(POLY_XLU_DISP++, 0, 0, 0, 0, 0, alpha);
-            gDPPipeSync(POLY_XLU_DISP++);
-            Matrix_Translate(dustEffect->pos.x, dustEffect->pos.y, dustEffect->pos.z, MTXMODE_NEW);
-            Matrix_ReplaceRotation(&play->billboardMtxF);
-            Matrix_Scale(dustEffect->xyScale, dustEffect->xyScale, 1.0f, MTXMODE_APPLY);
-            gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
-            index = 0.5f * dustEffect->timer;
-            gSPSegment(POLY_XLU_DISP++, 0x08, Lib_SegmentedToVirtual(sDustTextures[index]));
-            gSPDisplayList(POLY_XLU_DISP++, object_fr_DL_00B338);
+        if (!firstDone) {
+            POLY_XLU_DISP = Gfx_SetupDL(POLY_XLU_DISP, SETUPDL_0);
+            gSPDisplayList(POLY_XLU_DISP++, object_fr_DL_00B328);
+            gDPSetEnvColor(POLY_XLU_DISP++, 0, 0, 0, 0);
+            firstDone = true;
         }
+
+        alpha = (255.0f / 16.0f) * dustEffect->timer;
+        gDPSetPrimColor(POLY_XLU_DISP++, 0, 0, 0, 0, 0, alpha);
+        gDPPipeSync(POLY_XLU_DISP++);
+        Matrix_Translate(dustEffect->pos.x, dustEffect->pos.y, dustEffect->pos.z, MTXMODE_NEW);
+        Matrix_ReplaceRotation(&play->billboardMtxF);
+        Matrix_Scale(dustEffect->xyScale, dustEffect->xyScale, 1.0f, MTXMODE_APPLY);
+        gSPMatrix(POLY_XLU_DISP++, Matrix_NewMtx(play->state.gfxCtx), G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+        index = 0.5f * dustEffect->timer;
+        gSPSegment(POLY_XLU_DISP++, 0x08, Lib_SegmentedToVirtual(sDustTextures[index]));
+        gSPDisplayList(POLY_XLU_DISP++, object_fr_DL_00B338);
     }
     CLOSE_DISPS(play->state.gfxCtx);
```

Comment:
Maybe add a blank line here? Normally we have a blank line before `CLOSE_DISPS`, but with the brace here, I don't know what you think.

Reply:
Yeah, I've been putting a line before even after a brace. Just missed this one